localstack/services/firehose/firehose_api.py

```diff
@@ -97,6 +98,28 @@ def put_records(stream_name, records):
             except Exception as e:
                 LOG.error('Unable to put record to stream: %s %s' % (e, traceback.format_exc()))
                 raise e
+        if 'HttpEndpointDestinationDescription' in dest:
+            http_dest = dest['HttpEndpointDestinationDescription']
+            end_point = http_dest['EndpointConfiguration']
+            url = end_point['Url']
+            record_to_send = {
+                'requestId': str(uuid.uuid4()),
+                'timestamp': (int(time.time())),
+                'records': []
+            }
+            for record in records:
+                data = record.get('Data') or record.get('data')
+                record_to_send['records'].append({'data': data})
+            record_to_send_json = json.dumps(record_to_send)
+            headers = {
+                'Content-Type': 'application/json',
+                'Content-Length': str(len(record_to_send_json))
+            }
+            try:
+                requests.post(url, json=record_to_send, headers=headers)
+            except Exception as e:
+                LOG.error('Unable to put record to HttpEndpoint: %s %s' % (e, traceback.format_exc()))
```

Comment:
I'd probably make this an `INFO` level message (`ERROR` should only be used for very critical runtime system errors), and also include the `url`:
```
LOG.info('Unable to put Firehose records to HTTP endpoint %s: %s %s' % (url, e, traceback.format_exc()))
```